lib/cryptoexchange/exchanges/vinex/services/pairs.rb

```diff
@@ -0,0 +1,25 @@
+module Cryptoexchange::Exchanges
+  module Vinex
+    module Services
+      class Pairs < Cryptoexchange::Services::Pairs
+        PAIRS_URL = "#{Cryptoexchange::Exchanges::Vinex::Market::API_URL}/markets"
+
+        def fetch
+          output = super
+          adapt(output)
+        end
+
+        def adapt(output)
+          output["data"].map do |output|
+            base, target = output['symbol'].split("_")
```

Comment:
In the example response
```
BTC_VEIL
```

Our convention is that base currency is always the varying currency.

You want to reverse this. Refer to https://github.com/coingecko/cryptoexchange/wiki/Implementing-a-New-Exchange-(HTTP-API)#some-things-to-take-note